ocs_ci/ocs/utils.py

```diff
@@ -1012,3 +1013,24 @@ def revive_osd_external(ceph_cluster, osd_id):
         except CommandFailed:
             log.error("Failed to revive osd")
             raise
+
+
+def reboot_node(ceph_node, timeout=300):
+    """
+    Reboot a node with given ceph_node object
+
+    Args:
+        ceph_node (CephNode): ceph node object representing the node
+        timeout (int): Wait time for node to comeback
+
+    """
+    ceph_node.exec_command(
+        cmd='reboot',
+        check_ec=False,
+        long_running=True
+    )
+
+    try:
+        ceph_node.connect(timeout)
+    except SSHException:
+        log.exception(f"Failed to connect to node {ceph_node.hostname}")
```

Comment:
Don't we want to re-raise the exception
```suggestion
        log.exception(f"Failed to connect to node {ceph_node.hostname}")
        raise
```

Or raise some specific exception which we will define in the docstring? No need for extra exception and we can document maybe directly SSHExceptin if we really want to raise an exception. But I think that function should raise an exception or return boolean based on success/failure.

Reply:
done